extensions/azurePublishNew/src/components/provisioningWizards/CreateResourcesWizard.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import formatMessage from 'format-message';
+import { usePublishApi } from '@bfc/extension-client';
+
+import { userInfoState } from '../../recoilModel/atoms/resourceConfigurationState';
+import { Wizard, WizardStep } from '../shared/wizard/Wizard';
+
+import { WizardFooterWithUserPersona } from './footers/WizardFooterWithUserPersona';
+import { CreateActionContentStep, ChooseResourcesStep, ResourceConfigurationStep, ReviewResourcesStep } from './steps';
+
+type Props = {
+  onStepChange: (stepId: string) => void;
+  stepIndex?: number;
+};
+
+export const CreateResourcesWizard = React.memo((props: Props) => {
+  const { stepIndex, onStepChange } = props;
+  const userInfo = useRecoilValue(userInfoState);
+  const [steps, setSteps] = React.useState<WizardStep[]>([]);
+  const [isValidResourceConfiguration, setIsValidResourceConfiguration] = useState<boolean>(false);
+  const { setTitle, onBack } = usePublishApi();
+
+  const setDialogTitle = (step: WizardStep) => {
+    step && setTitle({ title: step.title, subText: step.subTitle });
+    return <></>;
```

Comment:
what is this for?

Reply:
Extensions have a different way of setting the dialog title, when ever the header is rendered, setDialogTitle() will override the default wizard header behavior.